### PR TITLE
Alternate power inlet housing for V0 (Astrodyne 082S or 082SM)

### PR DESCRIPTION
**Pull request by wrex**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [ ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?

New C14 power inlet housing for a different vendor (Astrodyne sourced from Mouser electronics).

#### How was it tested?

Sourced the power inlet from Mouser, printed the part, then:

1. Ensured the inlet snapped into place securely.
2. Installling a heatset insert to ensure the hole was sized properly.
3. Ensuring a 3mm screw fit into the hole made for it.
4. Ensuring the raised ridge fit into spec LDO rails.
5. Ensuring that all features were located appropriately by fitting into the v0.1 I'm currently building.

#### Any background context you want to provide?

The spec Adam Tech part has been on back order from Digikey for weeks. I've got one on order, but made this equivalent while waiting.

#### Screenshots (if appropriate)

See the README.md and images folder (the in-situ print with inlet installed is shown).

#### Further notes

I've used git and github for personal projects for several years, but this is my first actual PR. Slap me if I've done something silly.

### CODE REVIEW

**Comment by wrex** [comment]
Forgot to mention that in addition to being slightly larger, the biggest difference from the official inlet is that this Astrodyne is fully enclosed and has differently shaped clips on the side vs. top/bottom. This prevents the screws from attaching from the top as in the original. I had to create different tabs on the side as well as an "ear" for the screw. 

All things considered, I think I like this inlet and my housing better than the official ones. (Medical grade, fully metal-enclosed inlets are nice.)

File: printer_mods/README.md [hunk 1]
```diff
@@ -214,6 +214,7 @@ like so:
 | warriorofwire | [Slenderfan for Afterburner](./warriorofwire/slenderfan) | Lighter, quieter, smaller Afterburner | :x: | :x: | :heavy_check_mark: |:x:|
 | Weaslus | [Electronics Bay and Enclosure Seperator](./Weaslus/Electronics-Bay-Seperator) | Better enclose chamber and keep heat from electronics | :heavy_check_mark: | :x: | :x: |:x:|
 | Wile.e | [Deck Panel Support Clips](./wile-e1/Deck_Panel_Support_Clips) | Clips to help support lower deck panels to prevent (or fix) warping: | :grey_question: | :heavy_check_mark: | :heavy_check_mark: |:grey_question:|
+| wrez | [V0 Alternate IEC 320 C14 Power Inlet Housing](./wrez/astrodyne-inlet) | Drop in replacement for alternate-source power inlet if spec inlet is back-ordered | :heavy_check_mark: | :x: | :x: |
```
Comment (KiloQubit) [inline printer_mods/README.md:217]: Please add an :x: for the Switchwire column.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!  I just have a minor fix for the README below.

File: printer_mods/README.md (hunk 1 above)
Comment (wrex) [reply printer_mods/README.md]: Done! Thanks.

**Review by KiloQubit — APPROVED** [review]
Thanks for the update!

**Pull request merged**